README.md

```diff
@@ -5,9 +5,12 @@
 
 [![Badge License]][License]    
 [![Badge Guidelines]][Guidelines]    
-[![Badge Retired]][Retired]    
+[![Badge Retired]][Retired]
```

Comment:
Let's remove this.

Reply:
Well I'd like to try one of @DeeDeeG 's replacement options instead. Just for the time being preferably the sunset option with the choice to replace it with the retired version there after the fact. At least until the rebrand is complete because I know that will take a few months. As a courtesy to upstream(since they are publishing some fixes and accepting again for a bit) I feel like this is probably the best option for now.